<ui version="4.0" >
 <author>Zack Rusin &lt;zack@kde.org></author>
 <comment>Licensed under GNU LGPL</comment>
 <exportmacro></exportmacro>
 <class>SonnetConfigUI</class>
 <widget class="QWidget" name="SonnetConfigUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>441</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" >
    <widget class="KComboBox" name="m_langCombo" />
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox1" >
     <property name="title" >
      <string>Options</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="m_bgSpellCB" >
        <property name="text" >
         <string>Enable &amp;background spellchecking</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="m_skipUpperCB" >
        <property name="text" >
         <string>Skip all &amp;uppercase words</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="m_skipRunTogetherCB" >
        <property name="text" >
         <string>S&amp;kip run-together words</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="textLabel1" >
     <property name="text" >
      <string>Default language:</string>
     </property>
     <property name="buddy" >
      <cstring>kComboBox1</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="KEditListBox" name="m_ignoreListBox" />
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header>keditlistbox.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
